substrate/frame/broker/src/utility_impls.rs

```diff
@@ -138,4 +137,9 @@ impl<T: Config> Pallet<T> {
 
 		Ok(Some((region_id, region)))
 	}
+
+	/// Returns the current block number of Relay Chain
+	pub fn relay_height() -> RelayBlockNumberOf<T> {
+		<<<T as crate::Config>::Coretime as CoretimeInterface>::RelayChainBlockNumberProvider as BlockNumberProvider>::current_block_number()
+	}
```

Comment:
Instead of the new function why not use `RCBlockNumberProviderOf::<T::Coretime>::current_block_number();` anywhere that it's needed? This is already used in a few places in the pallet and is easier to reason about wherever it's used

Reply:
Removed